docs_devel/docs/93.BuildingInstallerPackage.md

```diff
@@ -132,75 +120,77 @@ OmegaT project prepares tasks to sign to packages for macOS.
+Use one of the following tasks to prepare the package directory
+structure in a `build/install/` folder:
 
 - installArmMacDist
 - installMacDist
 
-## Building distribution with Linux Java runtime
+## Building the distribution with a Linux Java runtime
 
-OmegaT has tasks to build a distribution with Java runtime for Linux OS.
+OmegaT has tasks to build a distribution with Java runtime for Linux:
 
 - linux64DistTarBz
 - linuxArm64DistTarBz
 
-These tasks build `tar.bz2` distribution package with JRE.
+These tasks build a `tar.bz2` distribution package that includes a JRE.
 
-## Java Runtime to bundle with
+## Bundled Java Runtime Environments
 
-You need to put Java Runtime package in a directory `assetDir` property specified.
-It is a parent directory of OmegaT project source in default.
+You need to put a Java Runtime package in the directory specified by the `assetDir` property. By default, it is the parent directory of the OmegaT project source directory.
 
-Following packages are recognized as a windows JRE.
+The following packages are recognized as windows JREs:
 
 - OpenJDK17U-jre_x86-32_windows_hotspot_17.0.8.1_1.zip
 - OpenJDK17U-jre_x64_windows_hotspot_17.0.8.1_1.zip
 
-Following packages are recognized as a macOS JRE.
+The following packages are recognized as macOS JREs:
 
 - OpenJDK17U-jre_aarch64_mac_hotspot_17.0.8.1_1.tar.gz
 - OpenJDK17U-jre_x64_mac_hotspot_17.0.8.1_1.tar.gz
 
-Following packages are recognized as a Linux JRE.
+The following packages are recognized as Linux JREs:
 
 - OpenJDK17U-jre_aarch64_linux_hotspot_17.0.8.1_1.tar.gz
 - OpenJDK17U-jre_x64_linux_hotspot_17.0.8.1_1.tar.gz
 
 
 ## Building Linux native packages
 
-OmegaT has tasks to generate Linux native packages in "DEB" and "RPM".
-We use `jpackage` packaging tool which is added in a Java Development Kit.
-It has been introduced from Java 14 and OmegaT project requires JDK 17.
+OmegaT has tasks to generate Linux native packages in "DEB" and "RPM" archive formats.
+We use the `jpackage` packaging tool which is provided by Java Development Kits.
+The tool has been introduced in Java 14. OmegaT project requires JDK 17.
 
 The packaging tool `jpackage` generates installable packages for Java applications.
-You can provide platform-specific packages for Linux, macOS and Windows to users
-with a familiar way to install and launch your applications.
+You can provide platform-specific packages for Linux, macOS and Windows to users that come
+with familiar ways to install and launch the packaged applications.
 
-`jpackage` bundles java runtime which is used to run Gradle or is used to compile a project
-that has `jpackage` utility. Because OmegaT project uses Java 11 to compile, you need to
+`jpackage` bundles the java runtime which is used to run Gradle or the one which is used to compile the project
+that has the `jpackage` utility. Because OmegaT project uses Java 11 to compile, you need to
 run Gradle with JDK17.
+-> Why ???
```

Comment:
@miurahr 
Is JDK 17 required by the version of Gradle we use?

Reply:
`jpackage` is a JDK feature that is general available in Java 16 and later.
See https://openjdk.org/jeps/392

JEP is [Java Enhancement Proposal](https://en.wikipedia.org/wiki/JDK_Enhancement_Proposal), that is java committer's specification note.

It is why deb/rpm package build requires Java 17 (LTS) or later.

Reply:
I think contribution manual already explain why.
> OmegaT has tasks to generate Linux native packages in "DEB" and "RPM" archive formats.
> We use the `jpackage` packaging tool which is provided by Java Development Kits.
> The tool has been introduced in Java 14. OmegaT project requires JDK 17.

Reply:
OK. So packaging an OmegaT release requires JDK 17, but compiling the application itself only requires JDK 11.

Reply:
That was not clear when I read the document. Would you mind rewording it so that it is clearer?

Reply:
I just committed a proposed rewording.